add metrec: arabic poetry dataset (#893)
huggingface/datasets: datasets/metrec/README.md

> ---

Reviewer: really nice !

annotations_creators:
- no-annotation
language_creators:
- found
languages:
- ar
licenses:
- unknown
multilinguality:
- monolingual
size_categories:
- 55K
source_datasets:
- original

> task_categories:
> - poetry_classification
> task_ids:

Reviewer: ```suggestion
task_categories:
- text-classification
task_ids:
- text-classification-other-poetry-classification
```

please use the tagger https://github.com/huggingface/datasets-tagging to see which categories are available

- **Point of Contact:** [Zaid Alyafeai](mailto:alyafey22@gmail.com)

### Dataset Summary

The dataset contains the verses and their corresponding meter classes.
Meter classes are represented as numbers from 0 to 13. 
The dataset can be highly useful for further research in order to improve the field of Arabic poems’ meter classification.
The train dataset contains 47,124 records and the test dataset contains 8,316 records.

### Supported Tasks and Leaderboards

The dataset was published on this [paper](https://www.sciencedirect.com/science/article/pii/S2352340920313792). A benchmark is acheived on this [paper](https://www.sciencedirect.com/science/article/pii/S016786552030204X).

### Languages

The dataset is based on Arabic.

## Dataset Structure

### Data Instances

A typical data point comprises a label which is out of 13 classes and a verse part of poem. 

### Data Fields

[N/A]

### Data Splits

The data is split into a training and testing. The split is organized as the following 

|           | Tain   | Test |
|---------- | ------ | ---- |
|data split | 47,124 | 8,316|

## Dataset Creation

> ### Curation Rationale
> 
> [N/A]
> 
> ### Source Data
> 
> [N/A]
> 
> #### Initial Data Collection and Normalization
> 
> The dataset was collected from [Aldiwan](https://www.aldiwan.net/).
> 
> #### Who are the source language producers?
> 
> The poems are from different poets.
> 
> ### Annotations
> 
> The dataset does not contain any additional annotations.
> 
> #### Annotation process
> 
> [N/A]
> 
> #### Who are the annotators?
> 
> [N/A]
> 
> ### Personal and Sensitive Information
> 
> [N/A]
> 
> ## Considerations for Using the Data
> 
> ### Discussion of Social Impact and Biases
> 
> [N/A]
> 
> ### Other Known Limitations
> 
> [N/A]
> 
> ## Additional Information
> 
> ### Dataset Curators
> 
> [N/A]
> 
> ### Licensing Information
> 
> [N/A]
> 
> ### Citation Information
> 
> [N/A]

Reviewer: ```suggestion
### Curation Rationale

[More Information Needed]

### Source Data

[More Information Needed]

#### Initial Data Collection and Normalization

The dataset was collected from [Aldiwan](https://www.aldiwan.net/).

#### Who are the source language producers?

The poems are from different poets.

### Annotations

The dataset does not contain any additional annotations.

#### Annotation process

[More Information Needed]

#### Who are the annotators?

[More Information Needed]

### Personal and Sensitive Information

[More Information Needed]

## Considerations for Using the Data

### Discussion of Social Impact and Biases

[More Information Needed]

### Other Known Limitations

[More Information Needed]

## Additional Information

### Dataset Curators

[More Information Needed]

### Licensing Information

[More Information Needed]

### Citation Information

[More Information Needed]
```
